### PR TITLE
0210 docs: fix bridge status

### PR DESCRIPTION
**Pull request by zmstone**
fix bridge status descriptions.

### CODE REVIEW

**Review by thalesmg — APPROVED** [review]

File: apps/emqx_bridge/i18n/emqx_bridge_schema.conf [hunk 1]
```diff
@@ -35,8 +35,18 @@ emqx_bridge_schema {
...
     desc_status {
                    desc {
-                         en: """The status of the bridge"""
-                         zh: """Bridge 的状态"""
+                         en: """The status of the bridge<br/>
+- <code>connecting</code>: the initial state before any health probes were made.<br/>
+- <code>connected</code>: when the bridge passes the health probes.<br/>
+- <code>disconnected</code>: when the bridge can not pass health probes.<br/>
+- <code>stopped</code>: when the bridge resource is requested to be stopped<br/>
+- <code>inconsistent</code>: When not all the nodes are at the same status"""
```
Comment (ieQu1) [inline apps/emqx_bridge/i18n/emqx_bridge_schema.conf:43]: Nit: No full stop at the end of the sentence.

File: scripts/test/start-two-nodes-in-docker.sh [hunk 2]
```diff
@@ -84,14 +85,17 @@ defaults
...
-   option tcplog
-   bind *:18083
-   default_backend emqx_dashboard_back
+    mode tcp
+    option tcplog
+    bind *:18083
+    default_backend emqx_dashboard_back
 
 backend emqx_dashboard_back
+    # must use a consistent dispatch when EMQX is running on different versions
```
Comment (ieQu1) [inline scripts/test/start-two-nodes-in-docker.sh:94]:
```suggestion
    # Must use a consistent dispatch when EMQX is running on different versions
```

**Review by ieQu1 — APPROVED** [review]

File: Makefile [hunk 3]
```diff
@@ -222,6 +222,9 @@ run: $(PROFILE) quickrun
 quickrun:
 	./_build/$(PROFILE)/rel/emqx/bin/emqx console
 
+docker:
+	@$(BUILD) $(PROFILE) docker
```
Comment (id) [inline Makefile:226]: ?
```suggestion
	@$(BUILD) $(PROFILE)-docker
```

Comment (id) [reply Makefile:226]: so that we keep the logic in one place

**Review by id — APPROVED** [review]

**Review by id — APPROVED** [review]

**Review by thalesmg — APPROVED** [review]

**Pull request merged**